auslib/web/views/client.py

```diff
@@ -62,7 +62,7 @@ def get(self, **url):
                     product_query = query.copy()
                     product_query["product"] = product
                     response_release, response_update_type = AUS.evaluateRules(product_query)
-                    if product_query['buildTarget'] not in response_release['platforms'].keys():
+                    if not response_release or not response_release.shouldServeUpdate(product_query):
```

Comment:
Lokos like we don't need the additional call to shouldServeUpdate, since it already gets called as part of evaluateRules (https://github.com/mozilla/balrog/blob/master/auslib/AUS.py#L90), so the "if not response_release" is enough here. Other than that, I think this is ready to go!